AmazonChimeSDK/AmazonChimeSDK/session/MeetingSessionStatusCode.swift

```diff
@@ -54,6 +54,12 @@ import Foundation
     /// When maximum concurrent video channel reached
     case videoAtCapacityViewOnly = 206
 
+    /// Designated input device is not responding and timed out.
+    case audioInputDeviceNotResponding = 82
```

Comment:
How come the values are different between Android and iOS?

Reply:
It seems like implementation detail where we have independent status codes assigned in the MeetingSessionStatus on Android but iOS just reflects the actual codes.